sormas-api/src/main/resources/strings.properties

```diff
@@ -1156,8 +1156,11 @@ messageBulkLinkEventHint = All selected cases will be linked to the selected eve
 messageInfrastructureLocked = Infrastructure data is locked and cannot be added/edited/imported/de-archived
 messageDeleteImmunizationVaccinations = Changing the means of immunization will delete all vaccinations that have been added to this immunization. Are you sure you want to proceed with the change?
 messageVaccinationNotRelevantForCase = This persons vaccination is not relevant for this case because the vaccination date is set after the symptom onset date or case report date.
+messageVaccinationNoDateNotRelevantForCase = This persons vaccination is not relevant for this case because it does not have a vaccination date.
 messageVaccinationNotRelevantForContact = This persons vaccination is not relevant for this contact because the vaccination date is set after the last contact date or contact report date.
+messageVaccinationNoDateNotRelevantForContact = This persons vaccination is not relevant for this contact because  it does not have a vaccination date.
```

Comment:
```suggestion
messageVaccinationNoDateNotRelevantForContact = This vaccination is not relevant for this contact because it does not have a vaccination date.
```